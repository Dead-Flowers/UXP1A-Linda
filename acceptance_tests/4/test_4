-t 1 output '(\"abcd\", 1440)'
-t 1 input '(string:*, integer:>1000)'
-t 1 read '(string:*, integer:*)'
