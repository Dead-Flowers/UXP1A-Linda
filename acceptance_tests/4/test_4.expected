Exit code: 0
("abcd",1440)
Exit code: 0
Exit code: 2
